template_ws/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM ros:humble
+LABEL org.opencontainers.image.authors="yuzhong1214@gmail.com"
+ARG USERNAME=user
+ARG USER_UID=1000
+ARG USER_GID=$USER_UID
+
```

Comment:
I'm not sure if the Timezone settings should be added back here.

https://github.com/j3soon/ros2-agv-essentials/blob/0c106004401fc5f9f7a62b3e646c02dc8b71f0dc/template_ws/ros2_ws/.devcontainer/Dockerfile#L44-L52